### PR TITLE
[GitIgnore] add seatunnel-examples from gitignore

### PR DESCRIPTION
**Pull request by EricJoy2048**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -47,3 +47,5 @@ Test.scala
 test.conf
 spark-warehouse
 *.flattened-pom.xml
+
+seatunnel-examples
```
Comment (hailin0) [inline .gitignore:51]: Need to end with `/`?

Comment (EricJoy2048) [reply .gitignore:51]: > Need to end with `/`?

Doesn't seem necessary.

**Review by hailin0 — APPROVED** [review]

**Comment by hk-lrzy** [comment]
LGTM

**Review by wuchunfu — APPROVED** [review]

**Review by nielifeng — APPROVED** [review]
LGTM

**Pull request merged**